content/download/lts/index.adoc

```diff
@@ -92,7 +92,9 @@ See the link:/content/event-calendar[event calendar] for the specific LTS RC/rel
 
 ## Backporting Process
 
-Any user can propose that a bug fix be backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+Any user can propose that a issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface.
+Additionally library updates can also be considered if the library contains a public CVE that does not affect Jenkins (if Jenkins is vulnerable via a library then the link:/content/security[security] process applies).
```

Comment:
@daniel-beck is your concern mainly around the second sentence here?

would the following be preferable to you (which completly oimits security and bug and just says "fix"?

```suggestion
For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix with no change to API surface.
```
thus we are not saying bug or improvement or library update just "fix" which can be (ambiguously) used to match dependency updates as well as bug fixes?

Reply:
at which point this adds nothing over https://github.com/jenkins-infra/jenkins.io/pull/4547/files#diff-a9b67297305fb5c7f11633ef364dbf7d56fe684ce1f73d94c722226a28517dc0R100

so we are left with the change from "bug fix" to "issue" in https://github.com/jenkins-infra/jenkins.io/pull/4547/files#diff-a9b67297305fb5c7f11633ef364dbf7d56fe684ce1f73d94c722226a28517dc0R95

Reply:
Sort of. (And given that vulnerabilities are bugs, even that is debatable 😉)

This is the public documentation for Jenkins users and as such has different goals than corresponding rules/guidelines for maintainers would have. It seems that you're seeing it as the latter when it is the former.